src/platform/imx8ulp/lib/memory.c

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2021 NXP
+//
+// Author: Daniel Baluta <daniel.baluta@nxp.com>
+
```

Comment:
Fix author here.

Reply:
But this file I haven't modified it, so does it also needs to change the author?

Reply:
Sure, but you added a new file. So at least add yourself to the authors.